components/eventing-controller/controllers/subscription/event.go

```diff
@@ -0,0 +1,33 @@
+package subscription
```

Comment:
I suggest to move this file under `components/eventing-controller/controllers/events/events.go`, as it could also be used by the `components/eventing-controller/controllers/backend/reconciler.go` in the future.